lib/xcknife/test_dumper.rb

```diff
@@ -291,7 +294,12 @@ def discover_tests_to_skip(test_bundle)
     end
 
     def simctl
-      @simctl_path
+      return @simctl_path unless @simctl_timeout > 0
+      return "#{gtimeout} #{@simctl_path}"
+    end
+
+    def gtimeout
```

Comment:
do we want to handle `gtimeout` not being installed?

Reply:
sure